.github/pull_request_template.md

```diff
@@ -1,14 +1,25 @@
 <!-- Remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm the following points: -->
 <!-- (it should look like this: - [x] I have ...) -->
+
 **_I affirm:_**
+
 - [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
 - [ ] I have read and understood the [Contributing Guide](https://github.com/AirSkyBoat/AirSkyBoat/blob/staging/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/AirSkyBoat/AirSkyBoat/blob/staging/CODE_OF_CONDUCT.md).
 - [ ] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.
 
-## What does this pull request do?
+## Please enter a customer-centric description
```

Comment:
I would phrase this as "player facing" to be more understandable to most people.